tests/components/openerz/test_sensor.py

```diff
@@ -0,0 +1,54 @@
+"""Tests for OpenERZ component."""
+from unittest.mock import MagicMock, patch
+
+from homeassistant.components.sensor import DOMAIN as SENSOR_DOMAIN
+from homeassistant.setup import async_setup_component
+
+MOCK_CONFIG = {
+    "sensor": {
+        "platform": "openerz",
+        "name": "test_name",
+        "zip": 1234,
+        "waste_type": "glass",
+    }
+}
+
+
+async def test_sensor_init(hass):
```

Comment:
I think we can remove this test. It's basically doing the same as the next test.